network-api/networkapi/wagtailpages/pagemodels/customblocks/listing.py

```diff
@@ -13,6 +16,22 @@ class ListingCard(blocks.StructBlock):
 
     link = blocks.PageChooserBlock(required=False, help_text="Page that this should link out to.")
```

Comment:
What do you think about calling this something like `link_page`?

Reply:
Thanks, changed in 470041f6164fa3dc18fb7a33ad8612ca82daae96